package.json

```diff
@@ -27,9 +27,9 @@
     "clean:tslint": "rimraf tslint-rules/*.js",
     "compile:tslint": "tsc -P tslint-rules",
     "compile:script": "tsc -P script/tsconfig.json",
-    "lint": "yarn lint:src && yarn lint:prettier",
-    "lint:fix": "yarn lint:src:fix && yarn lint:prettier --fix",
-    "lint:prettier": "ts-node script/prettier.ts",
+    "lint": "yarn prettier && yarn lint:src",
+    "lint:fix": "yarn prettier --write && yarn lint:src:fix",
+    "prettier": "prettier --check \"./**/*.{yaml,yml,scss,ts,tsx,js,jsx,html}\"",
```

Comment:
If file extensions are put in alphabetical order, it's easier to extend the list later by adding new extensions. Just a though from a personal experience, things don't have to be like this.